### PR TITLE
Issue #SH-1117 fix:  re-issue btn is enabled for svg

### PR DESCRIPTION
**Pull request by swetha-aggidevara**

### CODE REVIEW

File: src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.spec.ts [hunk 1]
```diff
@@ -459,4 +459,28 @@ describe('ReIssueCertificateComponent', () => {
     expect(component.unsubscribe$.next).toHaveBeenCalled();
     expect(component.unsubscribe$.complete).toHaveBeenCalled();
   });
+
+  it ('should return TRUE when the status is 2', () => {
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.spec.ts:463]: Update statement easy to understand.
'should enable "Re-Issue" button when status = 2 "

File: src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.spec.ts [hunk 2]
```diff
@@ -459,4 +459,28 @@ describe('ReIssueCertificateComponent', () => {
...
+    const batch = {batchId: '1', name: 'batch 1', certificates: [], ssuedCertificates: [], createdBy: '123', status: 2 };
+    const enableData = component.enableReIssueCert(batch);
+    expect(enableData).toBeTruthy();
+  });
+
+  it ('should return TRUE when the certificates are issued', () => {
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.spec.ts:469]: Update statement easy to understand.
'should enable "re-issue" button, when pdf certificate is issued("certificates" exisit)'

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.ts [hunk 3]
```diff
@@ -173,6 +173,11 @@ export class ReIssueCertificateComponent implements OnInit, OnDestroy {
     this.userName = this.userName ? this.userName.trim() : this.userName;
     const value = !_.isEmpty(this.userData) ? this.userData['courses'].batches = [] : [];
   }
+
+  enableReIssueCert(batch) {
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.ts:177]: What will happen when both PDF & SVG certificates are exist? Which certificate will get re-issues that time?
 
We should issue both SVG & PDF certificates. Check this once.

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.ts (hunk 3 above)
Comment (swetha-aggidevara) [reply src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.ts:177]: Certificate is being issued in this format:
SVG - SVG format
PDF - PDF format
SVG and PDF - SVG format

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**